docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc

```diff
@@ -76,6 +76,11 @@ This means that only a minimal amount of write operations is used to update the
 |Includes all modules, whose fully qualified name starts with the given prefix, in the synchronisation process.
 |===
 
+|`continueOnError`
+|Boolean
+|If the sync encounters an error, simply log it and continue instead of terminating. This most likely results in an incomplete sync result. Default to `false`.
+|===
```

Comment:
The table is broken because line 77 hasn't been removed.
Also the content does not fit the table header since it is not a method.

Reply:
fixed